src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -436,4 +436,13 @@
   <data name="UseParallelizeAttributeAnalyzerTitle" xml:space="preserve">
     <value>Explicitly enable or disable tests parallelization</value>
   </data>
+  <data name="TypeContaningTestMethodShouldBeATestClassTitle" xml:space="preserve">
+    <value>Type containing '[TestMethod]' must be a '[TestClass]'</value>
```

Comment:
```suggestion
    <value>Type containing '[TestMethod]' must be marked with '[TestClass]'</value>
```

Reply:
I am wondering if we should be not using the attribute name explicitely as we allow sub-classing, something like:

`Types containing test methods should be test classes`.

Reply:
Imho using the name `[TestClass]` is clearer for 90% of use cases. So I would keep it.

Reply:
Good for me. @engyebrahim please only apply the suggested change.